datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -94,21 +88,14 @@ def test_download(capfd, distribution_name, distribution_version, temporary_loca
 
 
 @pytest.mark.online
-@pytest.mark.skipif(PY2, reason="tuf builds for Python 2 do not provide required information in exception")
 def test_expired_metadata_error(distribution_name, distribution_version):
     """Test expiration of metadata raises an exception."""
     argv = [distribution_name, "--version", distribution_version]
 
-    # Make sure time.time returns futuristic time.
-    with mock.patch(
-        "time.time",
-        mock.MagicMock(return_value=time.mktime(datetime(year=2524, month=1, day=1).timetuple())),
-    ), pytest.raises(NoWorkingMirrorError) as exc:
+    # Make sure we use futuristic time.
+    with freeze_time("2524-01-01"), pytest.raises(ExpiredMetadataError):
```

Comment:
Nice: `freeze_time` and the new python-tuf API makes the test code more much more readable.